docs/io.cozy.contacts.md

```diff
@@ -6,6 +6,8 @@
 
 The `io.cozy.contacts` doctype is loosely based on the [vCard RFC](https://tools.ietf.org/html/rfc6350), in that some of the attributes have been renamed for clarity. The attributes with a `?` are optional.
 
+- `index` : {string} Field to be used to sort contacts. It's a concatenation of `givenName`, `familyName`, first `email` and `cozy.url`
+- `displayName` : {string} Name to be displayed. In order of priority, it's the `givenName` or `familyName` or first `email` or `cozy.url`
```

Comment:
```suggestion
- `displayName` : {string} Name to be displayed. In order of priority, it's the name, the email or the cozy URL
```

I don't think it's going to be givenName *or* familyName, we'll want both.

Reply:
It's `familyName` if `givenName` is empty. Actually it's `fullname` or first `email` or `cozy.url`

Reply:
I still find the description not very clear and probably too technical. Even my proposal is not that good in hindsight, I don't think we should talk about the implementation details in this doc. We may want to change the rules without changing the meaning of the field.

Reply:
Ok good point 👍

Reply:
Yeah we'll end up using it in drive as well. New description is good 👍  thanks